src/excmds.ts

```diff
@@ -961,6 +976,168 @@ export function jumpprev(n = 1) {
     })
 }
 
+/**
+ * Jumps to a local mark, a global mark, or the location before the last mark jump.
+ * [a-z] are local marks, [A-Z] are global marks and '`' is the location before the last mark jump.
+ * @param key the key associated with the mark
+ */
+//#content
+export async function jumpmark(key: string) {
+    if (key.length !== 1) {
+        return;
+    }
+    if (key === "`") {
+        return jumpBeforeJumpMark()
+    }
+    if (!/[a-z]/i.exec(key)) {
+        return;
+    }
+    if (key === key.toUpperCase()) {
+        return jumpGlobalMark(key);
+    }
+    return jumpLocalMark(key);
+}
+
+/**
+ * Jumps to a local mark.
+ * @param key the key associated with the mark
+ * @hidden
+ */
```

Comment:
Every function in excmds.ts should be explicitly annotated with `//#{content,background,both}{,_helper}`. I suspect setting marks should all be `//#content`, jumping to local marks should be `//#content`, but jumping to global marks should be `//#background`.

Reply:
I added //#content annotations to all the functions including markjumpglobal because it gets current url now.